src/docs/reference/modules/Sms/README.md

```diff
@@ -14,7 +14,7 @@ Enabling the `SMS` feature will add a new settings page under `Configurations` >
 !!! note
     After enabling the SMS feature, you must configure the default provider in order to send SMS messages.
 
-# Configuring Twilio Provider
+## Configuring the Twilio Providers
 
 To enable the [Twilio](https://www.twilio.com) provider, navigate to `Configurations` >> `Settings` >> `SMS`. Click on the `Twilio` tab, click the Enable checkbox and provider your Twilio account info. Then in the `Providers` tab, select Twilio as your default provider.
 
```

Comment:
Here you should add a note about the Azure Communication SMS feature under **Additional Providers** section with a link to the docs

Reply:
Ok

Reply:
@hishamco this was not yet added. Can you please add it?